packages/codemods/src/codemods/v2.x/updateApp/README.md

```diff
@@ -0,0 +1,5 @@
+# Update how we make the ClerkProvider context available to the app
+
+- Update imports
+- Remove old ClerkAuthConsumer and ClerkAuthProvider
+- Insert new ClerkAuthProvider implementation
```

Comment:
Does ClerkAuthProvider actually change? It didn't seem to when I ran it.

Reply:
How did you test this? It should change. One easy thing to spot should be that the new provider should use `<ClerkLoaded>` whereas the old one doesn't have that component at all.

Reply:
My mistake, I see it now.